db/migrate/20231121133009_fill_null_users_organisations_created_at.rb

```diff
@@ -0,0 +1,7 @@
+class FillNullUsersOrganisationsCreatedAt < ActiveRecord::Migration[7.0]
+  def change
+    UsersOrganisation.includes(:user).where(created_at: nil).find_each do |users_organisation|
```

Comment:
l'élément itérable devrait plutôt s’appeler `user_organisation` selon moi, mais c'est pas très important

Reply:
Oui sémantiquement tu as raison mais le nom du model est `users_organisation` donc je préfère rester fidèle au nom du modèle (on fait pareil avec les `rdvs_user` côté rdvs)